### DISCUSSION TITLE
Error when building vectors tile for North America w/ MAX_ZOOM=14

### DISCUSSION THREAD

**Discussion by diehl** [openmaptiles/openmaptiles#1669] [Q&A]
I'm trying to build vector tiles for North America that work up to zoom level 14. Last night, when running on my iMac Pro with 32 GB of RAM, I ran into the following error message. I'm not sure what to adjust to get around this problem. Any pointers would be greatly appreciated!!

> [2024-06-25T09:39:17Z] 7:36:58 [fatal] bulk insert into "osm_highway_linestring": SQL Error: pq: could not extend file "base/16384/60228.11": No space left on device in query COPY "import"."osm_highway_linestring" ("osm_id", "geometry", "highway", "construction", "tracktype", "ref", "network", "z_order", "layer", "level", "indoor", "name", "name_en", "name_de", "tags", "is_tunnel", "is_bridge", "is_ramp", "is_ford", "is_oneway", "is_area", "service", "access", "toll", "usage", "public_transport", "man_made", "bicycle", "foot", "horse", "mtb_scale", "sac_scale", "surface", "expressway") FROM STDIN

**Comment by ImreSamu** [comment]
> "No space left on device in query"

disk space problem.

**Reply by diehl** [reply]
@ImreSamu I thought that might be the case as well. Unless Docker freed up a bunch of disk space once the error occurred, I'm not seeing an issue. I have a 2 TB drive in the machine with 1 TB free. I imagine that's more than plenty of disk.

**Reply by ImreSamu** [reply]
* maybe you have to check/increase the disk space being allocated by Docker  ( via https://stackoverflow.com/a/68317889 )
* or check the Docker Desktop Mac FAQ .. https://docs.docker.com/desktop/faqs/macfaqs/#what-if-the-file-is-too-big

**Reply by diehl** [reply]
@ImreSamu I bumped up the virtual disk limit to 256 GB and am rerunning the process now to see if it completes. Found this idea here: https://github.com/openmaptiles/openmaptiles/issues/385
